sim/build.sbt

```diff
@@ -21,62 +21,36 @@ def isolateAllTests(tests: Seq[TestDefinition]) = tests map { test =>
 
 testGrouping in Test := isolateAllTests( (definedTests in Test).value )
 
-val rocketChipDir = file("target-rtl/firechip/rocket-chip")
-val fireChipDir  = file("target-rtl/firechip")
+lazy val firesimAsLibrary = sys.env.get("FIRESIM_STANDALONE") == None
```

Comment:
I think this is fine. I've used `sys.props` and `-D` options when invoking `sbt` b/c I thought that was a little more explicit, but environment vars are fine too.

Reply:
Ah, sounds good.